draft-ietf-taps-transport-security.md

```diff
@@ -796,16 +796,16 @@ The following table lists the availability of the above-listed optional features
 |---
 | Protocol | MA | CM | SV | AFN | CX | SC |
```

Comment:
The last column should be SM, to match the above? ("Session caching and management (SM)")

Reply:
I think SC makes more sense, but really it's just an abbreviation to make the table representable in ASCII: it doesn't really matter that much.